offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/AppView.groovy

```diff
@@ -51,8 +45,7 @@ class AppView extends VerticalLayout {
             OfferOverviewView overviewView,
             CreateOfferView updateOfferView,
             SearchPersonView searchPersonView,
-            MaintainProductsView maintainProductsView,
-            CreateProjectView createProjectView) {
```

Comment:
why remove this view?

Reply:
The create project view was used differently from all other views. It is not an entry point for the application but only exists in the context of the OfferOverviewView. Each OfferOverviewView requires its own CreateProjectView to work.